README.md

```diff
@@ -1,11 +1,53 @@
 OpenMower integration for HomeAssistant
 ===
 
+Openmower is the DIY RTK GPS Smart Mowing Robot for Everyone! Based on the YardForce classic 500 lawn mover and fully open source. A custom built motherboard, a raspberry pi and a GPS module gets rid of the guide wire and makes software the key factor in the project.
 
-Usage
+[openmower.de](https://openmower.de/)
+
+This integration is based on MQTT messages from the openmower container that runs on the RPI. 
+
+It will give you full control over the mower. Buttons to start, stop, pause, create automations from etc. Statistics like battery load, motor temperature, location. GPS coordinates are integrated into the device tracker.
+
+Prerequisits
+===
+* A functional openmower (doh!)
+* A software version where MQTT is supported, currently only on the edge version.
+* Activated MQTT in openmower config (/boot/openmower/mover_config.txt)
+```
+export OM_MQTT_ENABLE="True"            # Enable or disable
+export OM_MQTT_HOSTNAME="10.2.3.4"      # IP or hostname of your HA
+export OM_MQTT_PORT="1883"              # Port, default 1883
+export OM_MQTT_USER="mqtt_om"           # MQTT user on your HA
+export OM_MQTT_PASSWORD="mqtt_om"       # MQTT password on your HA
+export OM_MQTT_TOPIC_PREFIX="openmower" # The prefix that all MQTT traffic from this mower should have. If you have multiple mowers this can be used to separate them.
+```
+* Home assistant with HACS
+* Mosquitto or similar MQTT broker.
```

Comment:
maybe mention that addon is the simplest way to make it work? with a screenshot?